MapboxStatic.xcodeproj/project.pbxproj

```diff
@@ -908,57 +889,6 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
-/* Begin PBXShellScriptBuildPhase section */
-		DA9CC6901E5C570C00A14964 /* Copy Frameworks */ = {
```

Comment:
There is no more need to copy frameworks, path to framework is passed directly in `Link Binary with Library` build phase.